decidim-meetings/spec/lib/decidim/meetings/component_spec.rb

```diff
@@ -98,11 +98,23 @@
         create_list(:comment, 5, commentable: hidden_meeting)
       end
 
-      it "counts the comments from visible proposals" do
+      it "counts the comments from visible meetings" do
         expect(Decidim::Comments::Comment.count).to eq 8
         expect(subject).to eq 3
       end
     end
+
+    describe "attendees_count" do
+      let(:stats_name) { :attendees_count }
+      let!(:closed_meeting) { create(:meeting, :published, :closed, attendees_count: 20, component:) }
```

Comment:
Let's split this in two meetings so we can check out that the sum is being done
```suggestion
      let!(:a_closed_meeting) { create(:meeting, :published, :closed, attendees_count: 5, component:) }
      let!(:another_closed_meeting) { create(:meeting, :published, :closed, attendees_count: 15, component:) }
```